website_sale_product_subscribe_notify/models/product_template.py

```diff
@@ -24,19 +24,17 @@ def write(self, vals):
                     partner_ids = rec.get_product_category_followers_ids()
                     partners = rec.env['res.partner'].browse(partner_ids)
                     list_price = "%d" % int(rec.list_price)
-                    multi_partner_id = ''
-                    for partner in partners:
-                        multi_partner_id = multi_partner_id + str(partner.id) + ','
+                    limit_recipient = rec.get_limit_email_recipient()
                     ctx.update({
```

Comment:
The final step of this PR, is to loop this block of code based on the number we set in the configuration.